plugins/tag_search/services/TagService.php

```diff
@@ -34,18 +34,7 @@ public function searchAction(KalturaTagFilter $tagFilter, KalturaFilterPager $pa
         }
 
         $tagFilter->validate();
-
-        $c = KalturaCriteria::create(TagPeer::OM_CLASS);
-        $tagCoreFilter = new TagFilter();
-        $tagFilter->toObject($tagCoreFilter);
-        $c->setGroupByColumn('tag');
-        $tagCoreFilter->attachToCriteria($c);
-        $pager->attachToCriteria($c);
-        $tags = TagPeer::doSelect($c);
-
-        $searchResponse = new KalturaTagListResponse();
-        $searchResponse->objects = KalturaTagArray::fromDbArray($tags, $this->getResponseProfile());
-        $searchResponse->totalCount = $c->getRecordsCount();
+        $searchResponse = $tagFilter->getListResponse($pager, $this->getResponseProfile());
```

Comment:
can you explain this change?

Reply:
@inbal-ben-david  the tag functionality had not until now supported responseProfiles, which we now need it to do. So the code block previously implemented in tag->search moved onto the KalturaTagFilter, as with all of our more recent filters.
The tag feature was written originally before we had the responseProfile capability.